README.md

```diff
@@ -207,6 +111,17 @@ try {
 }
 ```
 
+### Customizing Style
+
+You can override the z-index using the pera-wallet-connect-modal class so that the modal does not conflict with another component on your dApp.
```

Comment:
```suggestion
You can override the z-index using the `.pera-wallet-connect-modal` class so that the modal does not conflict with another component on your application.
```